src/modules/form/components/ViewRecord.tsx

```diff
@@ -29,6 +29,7 @@ const ViewRecord = <RecordType extends SubmitFormAllowedTypes>({
 }: ViewRecordProps<RecordType>): JSX.Element => {
   const { formDefinition, itemMap, loading } = useFormDefinition({
     role: formRole,
+    id: 'formDefinitionId' in record ? record.formDefinitionId : undefined,
```

Comment:
Can you add a comment explaining this and that it comes from the form processor, so its the form that was last used to create/update the record